### PR TITLE
Update Files.hu-HU.xlf

### PR DESCRIPTION
**Pull request by KristofKekesi**
This PR is still in progress, please don't merge it yet.

### CODE REVIEW

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 1]
```diff
@@ -3176,55 +3176,55 @@
         </trans-unit>
         <trans-unit id="DecompressArchiveDialog.Title" translate="yes" xml:space="preserve">
           <source>Select a destination and extract files</source>
-          <target state="new">Select a destination and extract files</target>
+          <target state="translated">Válasszon egy célhelyet, és csomagolja ki a fájlokat</target>
         </trans-unit>
         <trans-unit id="DecompressArchiveDialogBrowseFolder.Content" translate="yes" xml:space="preserve">
           <source>Browse</source>
-          <target state="new">Browse</target>
+          <target state="translated">Keresés</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:3183]:
```suggestion
          <target state="translated">Kiválasztás</target>
```
Other alternatives: `Választás`, `Mappaválasztás`

This is a text of the button for opening folder picker dialog:

image

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 2]
```diff
@@ -3176,55 +3176,55 @@
...
         </trans-unit>
         <trans-unit id="DetailsArchiveItemCount" translate="yes" xml:space="preserve">
           <source>{0} items ({1} files, {2} folders)</source>
-          <target state="new">{0} items ({1} files, {2} folders)</target>
+          <target state="translated">{0} elem ({1} fájl, {2} mappa)</target>
         </trans-unit>
         <trans-unit id="PropertyUncompressedSize" translate="yes" xml:space="preserve">
           <source>Uncompressed size</source>
-          <target state="new">Uncompressed size</target>
+          <target state="translated">Tömörítettlen méret</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:3211]:
```suggestion
          <target state="translated">Tömörítetlen méret</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 3]
```diff
@@ -3308,141 +3308,141 @@
...
         </trans-unit>
         <trans-unit id="BaseLayoutItemContextFlyoutPinToFavorites.Text" translate="yes" xml:space="preserve">
           <source>Pin to Favorites</source>
-          <target state="new">Pin to Favorites</target>
+          <target state="translated">Kitűzés a kedvencekhez</target>
         </trans-unit>
         <trans-unit id="SideBarUnpinFromFavorites.Text" translate="yes" xml:space="preserve">
           <source>Unpin from Favorites</source>
-          <target state="new">Unpin from Favorites</target>
+          <target state="translated">Levétel a kedvencek közűl</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:3331]:
```suggestion
          <target state="translated">Levétel a kedvencek közül</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 4]
```diff
@@ -3308,141 +3308,141 @@
...
         </trans-unit>
         <trans-unit id="BaseLayoutContextFlyoutUnpinFromFavorites.Text" translate="yes" xml:space="preserve">
           <source>Unpin from Favorites</source>
-          <target state="new">Unpin from Favorites</target>
+          <target state="translated">Levétel a kedvencek közűl</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:3335]:
```suggestion
          <target state="translated">Levétel a kedvencek közül</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 5]
```diff
@@ -3308,141 +3308,141 @@
...
         </trans-unit>
         <trans-unit id="NavigationToolbarCopy.Label" translate="yes" xml:space="preserve">
           <source>Copy</source>
-          <target state="new">Copy</target>
+          <target state="translated">Másolás</target>
         </trans-unit>
         <trans-unit id="ConsentDialogTitle" translate="yes" xml:space="preserve">
           <source>Updates available</source>
-          <target state="new">Updates available</target>
+          <target state="translated">Frissítések elérhetőek</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:3355]:
```suggestion
          <target state="translated">Frissítések elérhetők</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 6]
```diff
@@ -3308,141 +3308,141 @@
...
         </trans-unit>
         <trans-unit id="ConsentDialogContent" translate="yes" xml:space="preserve">
           <source>Updates are ready to install</source>
-          <target state="new">Updates are ready to install</target>
+          <target state="translated">Frissítések elérhetőek a telepítésre</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:3359]:
```suggestion
          <target state="translated">Frissítések érhetők el telepítésre</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 7]
```diff
@@ -3044,7 +3044,7 @@
         </trans-unit>
         <trans-unit id="SecurityAppendDataLabel.Text" translate="yes" xml:space="preserve">
           <source>Append data</source>
-          <target state="new">Append data</target>
+          <target state="translated">Adatok hozzáadása</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:3047]:
```suggestion
          <target state="translated">Hozzáfűzés</target>
```

**Review by nvi9 — CHANGES_REQUESTED** [review]

**Comment by KristofKekesi** [comment]
Thanks @nvi9

**Review by gave92 — APPROVED** [review]

**Comment by gave92** [comment]
Sigh I've missed an issue in this resource file, project does not compile

**Pull request merged**